src/screens/settings/settings.tsx

```diff
@@ -128,35 +128,29 @@ export class SettingsScreenComponent extends React.Component<
                             <Switch
                                 onValueChange={async () => {
                                     try {
-                                        if (this.props.touchID) {
-                                            // disable touch id - delete pin
-                                            await clearPinCode();
-                                        } else {
-                                            this.props.openLoadingModal();
-                                            const password = await PasswordModal.getPassword(
-                                                undefined,
-                                                undefined,
-                                                {
-                                                    showCloseButton: true
-                                                }
-                                            );
-                                            await delay(0);
-                                            await setPinCode(password);
-                                        }
-                                        this.props.toggleTouchID();
+                                        this.props.openLoadingModal();
```

Comment:
i think you missed a flow here...
if biometric is active we have to disable it and remove pin code from keychain, no password modal.